frontend/packages/component-library/src/dropdown/__tests__/CustomDropdown.test.tsx

```diff
@@ -0,0 +1,169 @@
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import {CustomDropdown} from './../index';
+
+describe('Design System - Custom Dropdown Component', () => {
+  it('renders with correct text and children', () => {
+    render(
+      <CustomDropdown name="test-dropdown" labelText="Dropdown label" size="m">
+        <div data-testid="dropdown-content">Dropdown content</div>
+      </CustomDropdown>,
+    );
+
+    expect(screen.getByText('Dropdown label')).toBeInTheDocument();
+    expect(screen.getByTestId('dropdown-content')).toBeInTheDocument();
+  });
+
+  it('opens and closes when clicked', async () => {
+    const user = userEvent.setup();
+    render(
+      <CustomDropdown name="test-dropdown" labelText="Dropdown label" size="m">
+        <div data-testid="dropdown-content">Dropdown content</div>
+      </CustomDropdown>,
+    );
+
+    const dropdownButton = screen.getByRole('button', {
+      name: /test-dropdown filter dropdown/i,
+    });
+
+    // Initially dropdown content should not be visible
+    const dropdownContainer =
```

Comment:
please remove `data-testid="dropdown-content"` from this file tests as:
* looks like this specific test is the only one that uses test-it
* from what I know we've deprecated data-testID usage with eslinter from whole code-dot-org, not only our packages.
* for this specific test - won't it work if intsead of `screen.getByTestId('dropdown-content').parentElement;` we do smth like `screen.getByText('Dropdown content').parentElement;` ?

I don't mind keeping data-testid if there's no other way to implement this specific test (which I hope can be done without data-testid), for all the other tests let's please remove data-testid as it's not used (correct me if I'm wrong)